apps/evi/agent/lib/memory/store.test.ts

```
@@ -0,0 +1,53 @@
+import { drizzle } from 'drizzle-orm/postgres-js'
+import { describe, expect, it } from 'vitest'
+import { schema } from '../../../db/schema'
```

ESLint

[error] '../../../db/schema' imported multiple times.

(import/no-duplicates)